app/src/main/java/Controllers/ElasticsearchProblemController.java

```diff
@@ -63,79 +48,169 @@ public static void setClient(){
         }
     }
 
-    public static class DeleteAllProblemsForUserIDsTask extends AsyncTask<String, Void, Boolean>{
```

Comment:
should we still have DeleteAllProblemsForUserIDsTask that deletes all problem associated to a user (calls when user is deleted) or should we just combine DeleteProblemsTask and GetProblemsCreatedByUserIDTask?
Also need to consider the extension to records

Reply:
I totally missed that I had gotten rid of this! I'll add it back in.  I'll test it on a later pull request.

Reply:
This has been added in as of 5a69680